Objects/codeobject.c

```diff
@@ -339,6 +339,12 @@ init_code(PyCodeObject *co, struct _PyCodeConstructor *con)
     co->co_warmup = QUICKENING_INITIAL_WARMUP_VALUE;
     memcpy(_PyCode_CODE(co), PyBytes_AS_STRING(con->code),
            PyBytes_GET_SIZE(con->code));
+    int entry_point = 0;
+    while (_Py_OPCODE(_PyCode_CODE(co)[entry_point]) != RESUME &&
```

Comment:
Can this read the after code unit after the end of the buffer? Should the two operands of `&&` be swapped?